### PR TITLE
RuleSetNodeInstance should pass null variable values to DMN

### PR DESCRIPTION
**Pull request by SimonRinguette**
In the DMN engine, there is a difference between passing a variable with a null value and not passing a variable. If a DMN model would support receiving a null input. it is currently impossible to invoke this model from BPMN. The DMN model will complain about a missing input requirement (since the null value variable was not passed) and the execution will not append. With this fix, null value variables are passed to DMN and the execution works.

### CODE REVIEW

**Comment by etirelli** [comment]
jenkins test this

**Comment by etirelli** [comment]
@SimonRinguette any chance you have a unit test for this? 

The change absolutely makes sense, it is just that we need a unit test for each PR that gets merged.

**Comment by SimonRinguette** [comment]
Let me get something together

**Comment by SimonRinguette** [comment]
@etirelli There you go, without my fix, the test case fails. Its a simplified (as much as possible) way of reproducing it.

**Comment by mswiderski** [comment]
ok to test

File: jbpm-workitems/jbpm-workitems-bpmn2/src/test/java/org/jbpm/process/workitem/bpmn2/BusinessRuleTaskTest.java [hunk 1]
```diff
@@ -125,6 +125,27 @@ public void testDecisionTaskProcess() throws Exception {
...
+        params.put("Input",
+                   null);
+
+        WorkflowProcessInstance processInstance = (WorkflowProcessInstance) ksession.startProcess("passthru",
+                                                                                                  params);
+        Object output = processInstance.getVariable("Output");
+
+        assertEquals(null,
+                     output);
+    }
```
Comment (MarianMacik) [inline jbpm-workitems/jbpm-workitems-bpmn2/src/test/java/org/jbpm/process/workitem/bpmn2/BusinessRuleTaskTest.java]: Just to extend the coverage a little bit, would it be possible to add another test with non-null values? I think this kind of test is not present here yet.

Comment (SimonRinguette) [reply jbpm-workitems/jbpm-workitems-bpmn2/src/test/java/org/jbpm/process/workitem/bpmn2/BusinessRuleTaskTest.java]: I have added a simple test with a non-null value using the same model
